src/main/java/stream/flarebot/flarebot/util/pagination/PagedEmbedBuilder.java

```diff
@@ -65,13 +68,39 @@ public PagedEmbed build() {
         boolean pageCounts = false;
         if (list.getPages() > 1)
             pageCounts = true;
-        return new PagedEmbed(title, codeBlock, hasCodeBlock, list, pageCounts, color);
+        return new PagedEmbed(title, codeBlock, hasCodeBlock, list, pageCounts, color, groupPrefix, groupsPerPage);
     }
 
     public void setColor(Color color) {
         this.color = color;
     }
 
+    /**
+     * Enables the embed to use the groups within the {@link PaginationList}.
+     */
+    public void useGroups() {
+        this.groupPrefix = "";
+    }
+
+    /**
+     * Enables the embed to use the groups within the {@link PaginationList}. And sets the groups per page.
+     *
+     * @param groupsPerPage The amount of groups to put on a page.
+     */
+    public void useGroups(int groupsPerPage) {
+        this.groupPrefix = "";
```

Comment:
Still not sure why you set the groupPrefix to nothing.

Reply:
want me to add a boolean?

Reply:
No just whenever you use the shit, if it's null use a zero-width space.